Source/WebCore/page/scrolling/mac/ScrollerMac.h

```diff
@@ -27,6 +27,7 @@
 
 #if PLATFORM(MAC)
 
+#include "ScrollTypes.h"
 #include <WebCore/FloatPoint.h>
```

Comment:
Maybe we can forward declare `FloatPoint` as part of this cleanup?

Reply:
FloatPoint is referenced by value in the header.

Reply:
I only see `WebCore::FloatPoint convertFromContent(const WebCore::FloatPoint&) const;` in this header, and we can forward declare return value types. Is there some place I'm missing?

Also, it's a little strange to see `WebCore::` in the type and `WebCore/` in the include, given that this file is in WebCore (an existing issue).